package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "boom",
   "description": "HTTP-friendly error objects",
-  "version": "2.7.2",
+  "version": "2.8.0",
```

Comment:
Can you undue this? I do all the versioning stuff in one step with `npm version`.